spec/models/drift_state/purging_spec.rb

```diff
@@ -72,11 +72,6 @@
       end
     end
 
-    it "#purge_counts_for_remaining (used by tools - expensive, avoid)" do
-      expect(described_class.send(:purge_counts_for_remaining, 1))
-        .to eq(["VmOrTemplate", 1] => 1, ["VmOrTemplate", 2] => 2)
-    end
-
```

Comment:
@Fryguy The original code calls the "public" (less granular) test for `purge_count_for_remaining`. So we are testing the value `3` not the individual parts.
But there are tests that test the end result, so we do know which records remain the the db